src/inverseRobot.js

```diff
@@ -8,6 +8,21 @@
 
 function inverseRobot(robot) {
   // write code here
+  const keys = Object.values(robot);
+  const uniqueKeys = new Set(keys);
+
+  if (keys.length !== uniqueKeys.size) {
+    return null;
+  }
+
+  const values = Object.keys(robot);
+  const invertedRobot = {};
+
+  for (let i = 0; i < keys.length; i++) {
+    invertedRobot[keys[i]] = values[i];
+  }
```

Comment:
why do you need separate array for object keys? (and why did you name the variable `values`?)
you van iterate by keys using `for in` loop

Reply:
yes, i agree. It was strange solution) I also changed code, where null was returned